test/load-testing/README.md

```diff
@@ -0,0 +1,65 @@
+# How to Perform Load Testing
```

Comment:
Please add a note about updating the rate limit settings in the manifest YAML files and deploying them before running the tests.